src/main/java/seedu/address/model/internship/JobStatus.java

```diff
@@ -0,0 +1,36 @@
+package seedu.address.model.internship;
+
+public enum JobStatus {
```

Comment:
Perhaps we could name this `InternshipStatus` or something along those lines instead? Do let me know if I'm mistaken, but it seems that we have a Job class that has the variables - jobTitle, companyName and industry. So this status object should be reflective of an Internship status rather than a Job status? Shall we ensure that "Job" and "Internship" are not interchangeable terms throughout our code base? Another issue that might still arise even after changing the name to `InternshipStatus` is that we will then have a `Status` and `InternshipStatus` class, and it may not be obvious what the difference is between the 2 in terms of their names? Would it be possible/better to make this a private enum within the Status class?

Reply:
Yup, I think its best if we could ensure that the "Job" and "Internship" names are not used loosely throughout the code base. I don't particularly like the idea of a nested class, since I do intend to use the enum outside of this Status class (for parsing and input). Also, would it better to rename the `Status` class as `ApplicationStatus` and the enum to be called `Status` instead?

Reply:
Ah I see, thanks for the clarification! I think the `Status` and `ApplicationStatus` names suggested work!

Reply:
@seanjyjy @ZoroarkDarkrai @orzymandias Does anyone have any good suggestions? I'm leaning onto `Status` composing `ApplicationStatus` and `Date` atm.

Reply:
I think what Shawn said makes sense. It might be better to stick to terms that are already established eg `InternshipStatus`. If I am not wrong does the `status` class exist only for `Internship`? If that is the case, maybe you can consider shifting `date` to the `Internship` object instead and as such, there won't be any ambiguity in the naming.

Reply:
Thanks for the input. I think it does make sense and I have gone ahead and make the changes accordingly.